src/components/Avatar/Avatar.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+    import type { Action } from "svelte/action";
+    
+    // Note: consider `./types.ts`
+    interface AvatarProps {
+        // initials
+        initials?: string;
+        initialsFill?: string;
+        initialsFontSize?: number;
+        initialsBase?: string;
+        initialsRest?: string;
+
+        // image
+        imageBase?:string;
+        imageSrc?: string;
+        imageAlt?: string;
+        imageFallback?: string;
+        imageAction?: Action<HTMLElement, string>;
+        imageActionParams?: string;
+        imageRest?: string;
+
+        // figure
+        figureBase?: string;
+        figureBackground?: string;
```

Comment:
Using the Tailwind naming convention, this would be `rootBg` to support classes like `bg-green-500`. So far the only exception to Tailwind naming convention is single letter prefixes, like `p-4` and `m-4` (use padding/margin respectively)